draft-ietf-doh-dns-over-https-latest.mkd

```diff
@@ -407,6 +409,10 @@ In order to maximize interoperability, DNS API clients and DNS API
 servers MUST support the "application/dns-message" media
 type. Other media types MAY be used as defined by HTTP Content
 Negotiation ({{RFC7231}} Section 3.4).
+Those media types MUST be flexible enough to express every
```

Comment:
add a blank line to make a new para?

s/Those media/Media/  ?

Reply:
Should be in the same paragraph. I think "Those" is appropriate to tie this sentence to the preceding one.